src/js/events/events.js

```diff
@@ -66,6 +66,7 @@ define([], function() {
         JWPLAYER_CONTROLS: 'controls',
         JWPLAYER_FULLSCREEN: 'fullscreen',
         JWPLAYER_RESIZE: 'resize',
+        JWPLAYER_LOAD_PLAYLIST: 'loadPlaylist',
```

Comment:
No more new sysmbols here please. Use strings. I would like to deprecate jwplayer.events. from the player. They are long, repetitive and make the player bigger.